src/client/app/directives/directives.js

```diff
@@ -0,0 +1,18 @@
+interview_coding_app.directive('ngConfirmClick', [
+  function(){
+    return {
+      priority: -1,
+      restrict: 'A',
+      link: function(scope, element, attrs){
+        element.bind('click', function(e){
+          var message = attrs.ngConfirmClick;
+          // confirm() requires jQuery
+          if(message && !confirm(message)){
+            e.stopImmediatePropagation();
+            e.preventDefault();
+          }
+        });
+      }
+    }
+  }
```

Comment:
in this case I used an external resource for making confirmation dialogs, in case that we will use in other parts of our site, 

A simple solutions it is adding the confirmation dialog before in the function where the ng-click goes